Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -48,7 +46,7 @@ open class Player: UIViewController {
     }
 
     open var isFullscreen: Bool {
-        guard let core = self.core else {
+        guard let core = core else {
```

Comment:
Since you touched, you might as well make it a one liner. 
```suggestion
        core?.isFullscreen ?? false
```
On a side note, I wonder if for tvos, core.isFullscreen should default to `true` 🤔